src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerConstants.java

```diff
@@ -99,7 +99,8 @@ private CaseWorkerConstants() {
     public static final String FILE_MISSING_HEADERS = "The file is missing some column headers."
         + " Review and upload again";
 
-    public static final String NO_LOCATION_PRESENT = "You must add location details and upload the file again.";
+    public static final String NO_PRIMARY_LOCATION_PRESENT = "You must add Primary Base Location Name and"
```

Comment:
There should be space after `and`

Reply:
ah I missed this! thanks @SujeethKumarAri will amend it now